apps/examples/src/examples/speech-bubble/SpeechBubble/SpeechBubbleUtil.tsx

```diff
@@ -109,6 +109,7 @@ export class SpeechBubbleUtil extends ShapeUtil<SpeechBubbleShape> {
 			{
 				id: 'tail',
 				type: 'vertex',
+				label: 'Move tail',
```

Comment:
Shouldn't we be localizing things? Or do we only use english for a11y?

Reply:
this is apps/examples so it doesn't matter :P

Reply:
Ah, didn't see 😅